nima_core/scripts/migrate_themes.py

```
@@ -0,0 +1,111 @@
+    
+    for db_path in DB_PATHS:
+        if needs_themes_migration(db_path):
+            if add_themes_column(db_path, dry_run):
+                migrated.append(db_path)
+        else:
+            print(f"  ✅ {db_path.name}: already has themes column")
+    
+    print()
+    if migrated:
+        print(f"✅ Migration complete: {len(migrated)} databases updated")
+        print(f"   Run: python3 migrate_themes.py on all bots (Jac, Pinky, Knox, Phil, Atlas)")
```

Ruff (0.15.5)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)